app/src/actions/counter.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {mockStore} from '../mocks/mock-store';
+import {INCREMENT_COUNTER, DECREMENT_COUNTER} from './counter';
+import * as CounterActions from './counter';
+import {
+  describe,
+  it
+} from 'angular2/testing';
+
+describe('counter action creators', () => {
+  it('increment should create INCREMENT_COUNTER action', () => {
+    chai.expect(CounterActions.increment())
```

Comment:
from what I seen, 'angular2/testing' doesn't provide the usual jasmine 'assertions' (ie: expect(1).toBe(1)). including chai.expect related stuff works great in angular 2 (plus it was what we used for angular 1).


The big thing to remember is that we can't have both Jasmine and Mocha on the page at the same time.